etc/profile-a-l/gget.profile

```diff
@@ -0,0 +1,59 @@
+# Firejail profile for gget
+# Description:  a cli. to get things. from git repos
```

Comment:
```suggestion
# Description: a cli. to get things. from git repos
```

Reply:
Sorry, I just pushed the button when I saw this. Fixed in https://github.com/netblue30/firejail/commit/8a7b96974fd8ddae8ea4641d8e010b1e03b9e817. Thanks!